consensus/istanbul/core/roundchange.go

```diff
@@ -95,6 +175,7 @@ func (c *core) handleRoundChange(msg *message, src istanbul.Validator) error {
 	// try to catch up the round number.
 	if c.waitingForRoundChange && num == c.valSet.F()+1 {
 		if cv.Round.Cmp(roundView.Round) < 0 {
+			// TODO(asa): If we saw a PREPARED certificate, do we include one in our round change message?
```

Comment:
Wdyt @trianglesphere? Looks like I left a lot of open questions 🤦‍♂

Reply:
For this we actually do include that `prepared certificate`. Again I feel like it doesn't make a huge difference either way if it's included or not.